README.md

```diff
@@ -29,7 +29,7 @@ $ curl https://sirepo.run | bash
 
 For this to work, you will need to [install the prerequisites](https://github.com/radiasoft/download/blob/master/README.md#requirements).
 
-[API Documentation is available on Read the Docs.](http://sirepo.readthedocs.org)
+[API Documentation is available on Read the Docs.](https://sirepo.readthedocs.io)
 
 ### Manual Install with Docker
 
```

Comment:
Might as well update the copyright year on line 52

Reply:
2ad63f9b5f60e50c0cd8d0b67e643a3e756bda8e